IoT_master/tools/camera_initialize.py

```diff
@@ -0,0 +1,37 @@
+import cv2
+
+class Camera:
+    def __init__(self):
```

Comment:
Can you please add docstrings to your functions?
The format should be:
```
“””Short description of function’s purpose
with all necessary details for anyone
that will read this code in the future.

:param myvar: Description of myvar
:type myvar: Type of myvar
:param myvar2: Description of myvar2
:type myvar2: type of myvar2

:return: Description of what is returned
:rtype: returned type
“””
```

However, don’t describe self parameter

Reply:
Done.